tests/e2e/java_test.go

```diff
@@ -51,6 +51,32 @@ var _ = Describe("odoJavaE2e", func() {
 			runCmd("odo delete javaee-git-test -f")
 		})
 
+		It("Should be able to deploy a git repo that contains a wildfly application without wait flag", func() {
+
+			// Deploy the git repo / wildfly example
+			cmpCreateLog := runCmd("odo create wildfly wo-wait-javaee-git-test --git " + warGitRepo)
+			Expect(cmpCreateLog).Should(ContainSubstring("This may take few moments to be ready"))
+			dcName := runCmd("oc get dc | grep wo-wait-javaee-git-test | cut -f 1 -d ' '")
+			// following the logs and waiting for the build to finish
+			_ = runCmd("oc logs --version=1 dc/" + dcName)
```

Comment:
It won't handle if ```oc logs --version=1 dc/" + dcName``` error out. So try something like 
```
stdout, err = runCmd("oc logs --version=1 dc/" + dcName)
if err != nil {
		return fmt.Errorf("Error: %v", err)
	}
[...]
```
And change accordingly the ```runCmd()``` so that it returns stdout and stderr. For example
```
func runCmd(cmdS string) string, error {
[...]
return string(session.Out.Contents()), err
```

Reply:
error handling for `runCmd` is happening here https://github.com/redhat-developer/odo/blob/master/tests/e2e/e2e.go#L23
also if you are suggesting to modify the implementation of `runCmd` I think we can do it in a separate issue